src/reducers/breakpoints.js

```diff
@@ -92,7 +92,8 @@ function addBreakpoint(state, action) {
 
   // when the action completes, we can commit the breakpoint
   if (action.status === "done") {
-    return syncBreakpoint(state, action.value);
+    const { value } = ((action: any): DonePromiseAction);
+    return syncBreakpoint(state, value);
```

Comment:
could we move the type check into `syncBreakpoint`?

Reply:
We'd have to refactor a bit because the action `syncBreakpoint()` expects isn't a PromiseAction so the structures are different. In this case, this seems like the least intrusive approach but open to updating if you'd like.

Reply:
cool. I'm good with that